cookbooks/fb_ipset/recipes/default.rb

```diff
@@ -18,10 +18,6 @@
 # limitations under the License.
 #
 
-unless node.centos?
```

Comment:
Add debian and ubuntu to metadata as well; also, we should still fail on non-Linux platforms here